File: app/cs50x2025/content/spanish/lectures_source_code/src6/1/positive.c
// Abstracción y alcance

#include <cs50.h>
#include <stdio.h>

int get_positive_int(void);

int main(void)
{
    int i = get_positive_int();
    printf("%i\n", i);
}

// Le pide al usuario un entero positivo
int get_positive_int(void)
{
    int n;
    do
    {
        n = get_int("Entero Positivo: ");
    }
    while (n < 1);
    return n;
}

File: app/cs50x2025/content/spanish/lectures_source_code/src6/1/positive.py
# Abstracción y alcance

from cs50 import get_int


def main():
    i = get_positive_int()
    print(i)


# Solicita al usuario un número entero positivo
def get_positive_int():
    while True:
        n = get_int("Número entero positivo: ")
        if n > 0:
            break
    return n


main()